build.gradle

```diff
@@ -717,7 +746,7 @@ task distZip(type: Zip, dependsOn: [docsZip, schemaZip]) {
 		into "${baseDir}/schema"
 	}
 
-	subprojects.each { subproject ->
+	javaProjects.each { subproject ->
```

Comment:
@artembilan - copied from SI - is it correct that the bom is not included in `distZip.zip`?

Reply:
I don't think so.
Not sure, though, if anyone uses this zip, but for consistency it is probably better to include that BOM file as well.

Reply:
It doesn't seem to get staged anywhere, just in the .m2 repo with `publishToMavenLocal` so I don't see where to copy it from.

Let's just stick with this.

Reply:
See my solution in SI: https://github.com/spring-projects/spring-integration/pull/3940